docs/transformdata.rst

```diff
@@ -15,133 +15,141 @@ Transformation usage and configuration
 --------------------------------------
 
 *Integration Service* must be configured through its XML :ref:`configuration` file.
-In this case is needed a :ref:`Transformation Library` to provide *transformation functions* that will transform the data from *DDS World A* to *DDS World B* and vice versa.
+In this case, a :ref:`Transformation Library` is needed to provide *transformation functions* that will transform the data from *DDS World A* to *DDS World B* and vice versa.
 
+In this example, a file named :class:`config.xml` needs to be created.
 The *endpoints* must be configured in the :ref:`Fast-RTPS profiles` section.
 
 .. literalinclude:: config_comms_dds.xml
     :language: XML
     :start-after: <!-- fast-rtps profiles -->
     :end-before: <!-- end fast-rtps profiles -->
+    :dedent: 4
 
-And our needed :ref:`Connectors` are declared below:
+And the needed :ref:`Connectors` are declared below:
 
 .. literalinclude:: config_comms_dds.xml
     :language: XML
     :start-after: <!-- connectors -->
     :end-before: <!-- end connectors -->
+    :dedent: 4
```

Comment:
Where the transformation library is configured?

Reply:
We have added these lines to the configuration file